### PR TITLE
created some handlebars and uploaded images

### PR DESCRIPTION
**Pull request by rinoma**
Relates #16

### CODE REVIEW

**Review by tyeung2017 — APPROVED** [review]
nic pics

**Comment by rinoma** [comment]
OMG much wow

File: src/controllers/index.js [hunk 1]
```diff
@@ -11,15 +11,20 @@ const express = require('express');
...
 
+const home = require('./home');
+const cusines = require('./cusines');
+const error = require('./error');
 
 
-
-router.get('/', (req, res) => res.send('home'));
+router.get('/', home);
 router.get('/login', (req, res) => res.send('login'))
```
Comment (Karyum) [inline src/controllers/index.js:20]: what are these for ?

File: src/controllers/index.js [hunk 2]
```diff
@@ -11,15 +11,20 @@ const express = require('express');
 const path = require('path');
 const router = express.Router();
 
+const home = require('./home');
+const cusines = require('./cusines');
```
Comment (Karyum) [inline src/controllers/index.js:15]: this is not being used so why did you require it ?

**Review by Karyum — CHANGES_REQUESTED** [review]

File: src/controllers/index.js (hunk 2 above)
Comment (rinoma) [reply src/controllers/index.js:15]: we do now, in the next pull request

File: src/controllers/index.js (hunk 1 above)
Comment (rinoma) [reply src/controllers/index.js:20]: we were just working on these routes and now they work

Comment (rinoma) [reply src/controllers/index.js:20]: in the new pull request

**Review by Karyum — APPROVED** [review]

**Pull request merged**